src/test/regress/expected/local_shard_execution.out

```diff
@@ -1073,15 +1073,15 @@ BEGIN;
 	EXECUTE local_multi_row_insert_prepare_no_param;
 NOTICE:  executing the command locally: INSERT INTO local_shard_execution.distributed_table_1470001 AS citus_table_alias (key, value, age) VALUES (1,'55'::text,'21'::bigint), (5,'15'::text,'33'::bigint) ON CONFLICT(key) WHERE ((key OPERATOR(pg_catalog.>) 3) AND (key OPERATOR(pg_catalog.<) 4)) DO UPDATE SET value = ('88'::text OPERATOR(pg_catalog.||) excluded.value)
 	EXECUTE local_multi_row_insert_prepare_no_param;
-NOTICE:  executing the command locally: INSERT INTO local_shard_execution.distributed_table_1470001 AS citus_table_alias (key, value, age) VALUES (1,'55'::text,'21'::bigint), (5,'15'::text,'33'::bigint) ON CONFLICT(key) WHERE ((key OPERATOR(pg_catalog.>) 3) AND (key OPERATOR(pg_catalog.<) 4)) DO UPDATE SET value = ('88'::text OPERATOR(pg_catalog.||) excluded.value)
+NOTICE:  executing the command locally: <optimized out by local execution>
```

Comment:
Is it possible to do deparsing when logging? Or will we wait for other PRs to restore this message?

Reply:
Yes, if we use these changes in this PR too:
1. https://github.com/citusdata/citus/pull/3368/files#diff-063646ee3d261c4837768c512d35fff8R170-R176
2. https://github.com/citusdata/citus/pull/3368/files#diff-063646ee3d261c4837768c512d35fff8R659-R667

Reply:
I've done this now. My only minor concern was that the query that we show in the logs is not the exact same plan/query that we're executing (though should be the same anyway). But, that sounds OK I think.